app/helpers/show_summary_helper.rb

```diff
@@ -0,0 +1,40 @@
+# ShowSumamryHellper define handy methods such as check fields aviabaility
+module ShowSummaryHelper
+  def self.study_details?(presenter)
```

Comment:
@yf508 can you help? It looks that this function will always return true despite those fields are empty in some example record.

Reply:
As unless ShowSummaryHelper.study_details?(@presenter) is used in the view, the current method is a bit confusing, e.g. compared to the current return values, it should return the **opposite** value. e.g.

If the study_details needs to be displayed, return true, otherwise return false.